GenderPayGap.WebUI/Views/ChangePersonalDetails/ChangePersonalDetails.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Account.ChangePersonalDetailsViewModel
+
+@{
+    ViewBag.Title = "Change your personal details - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Change your personal details
+        </h1>
+            
+        <form method="POST" action="@Url.Action("ChangePersonalDetailsPost", "ChangePersonalDetails")">
+            @(Html.AntiForgeryToken())
+            
+            @(Html.GovUkErrorSummary())
+            
+            @(Html.GovUkTextInputFor(
+                m => m.FirstName,
+                labelOptions: new LabelViewModel { Text = "First name" },
+                autocomplete: "given-name"
+            ))
+            
+            @(Html.GovUkTextInputFor(
+                m => m.LastName,
+                labelOptions: new LabelViewModel { Text = "Last name" },
+                autocomplete: "family-name"
+                ))
+            
+            @(Html.GovUkTextInputFor(
+                m => m.JobTitle,
+                labelOptions: new LabelViewModel { Text = "Job title" },
+                autocomplete: "organization-title"
+                ))
+            
+            @(Html.GovUkTextInputFor(
+                m => m.ContactPhoneNumber,
+                labelOptions: new LabelViewModel { Text = "Contact phone number" },
+                autocomplete: "tel"
+                ))
+            
+            @(Html.GovUkButton(
+                new ButtonViewModel
+                {
+                    Text = "Confirm"
+                }
+                ))

```

Comment:
Quite often I add the class `govuk-!-margin-bottom-4` to these buttons, to slightly improve the spacing between the button and the cancel link under it
e.g.
```cshtml
@(Html.GovUkButton(new ButtonViewModel
{
    Text = "Continue",
    Classes = "govuk-!-margin-bottom-4"
}))
```